src/hardware/serialport/serialport.cpp

```diff
@@ -1306,7 +1306,7 @@ class SERIALPORTS final : public Module_base {
 			s_property[6] = '1' + static_cast<char>(i);
 			PropMultiVal* p = section->GetMultiVal(s_property);
 			std::string type = p->GetSection()->Get_string("type");
-			CommandLine cmd(0,p->GetSection()->Get_string("parameters"));
+			CommandLine cmd("", p->GetSection()->Get_string("parameters"));
```

Comment:
Awesome catch in cleaning up these "zero passed in place of a pointer" instances!

These "I'll just pass a zero in" are extremely misleading to readers.. Readers will see this as a measure of length (strcpy/snprintf) or a starting offset or absolute index.  Small but very impactful!